### DISCUSSION TITLE
Entering captcha manually

### DISCUSSION THREAD

**Discussion by Ababaandrey28** [PrismarineJS/mineflayer#3342] [Q&A]
In general, I need to implement a function somehow so that I can see the bot's screen In the first person so that I can manually enter a captcha for the bot

**Comment by rom1504** [comment]
where does the captcha appear

**Reply by Ababaandrey28** [reply]
in front of the player on the wall, you can say that paintings are placed through the frames that are combined into one image with text

**Reply by Ababaandrey28** [reply]
![1686399369366 » Leaked-Minecraft pro «](https://github.com/PrismarineJS/mineflayer/assets/165599766/a97d51a0-519a-4036-ae79-9ae81d63a735)
here, in the photo, this captcha is in front of the player, I just need to see the first-person view of the bot and so that I can enter the captcha into the console and it went for the name of the bot

**Reply by Vyakoo** [reply]
prismarine-viewer allows you to see through the browser what the bot sees, but I'm not sure about something complex like captcha https://github.com/PrismarineJS/prismarine-viewer

**Comment by greenik228** [comment]
I tried to implement this function using [mineflayer-item-map-downloader](https://github.com/IceTank/mineflayer-item-map-downloader), but I was faced with the fact that some of the images were flipped to the wrong position
